### ISSUE TITLE
What should happen & when if a business_bceid_name changes

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#2549]
#### Describe the task

Currently, it's not possible to change the `bceid_business_name` in the app.  When we create a user, we get `bceid_business_name` from the session in the FE and send it to the endpoint to put into the db. When we update a user, we send the same information from the FE, but the `UserUpdateIn` ninja schema strips out session. So the db never receives the new business name.

If someone needed to change their BCEID business name, the process would look ~something like this:
- user contacts Service BC to get the business name changed for the BCEID
- we put `bceid_business_name` into `UserUpdateIn`
- user logs into the app once Service BC has changed the name
- user goes to the profile page and submits. There's no field for business BCEID name that they can change, but when they submit, that sends the session info and their name will be updated
- BCEID business name should now be updated in the app (check by looking at the user access requests grid as a CAS user)

#### Acceptance Criteria

- [ ] first
- [ ] second
- [ ] third

#### Definition of Done

- [ ] first
- [ ] second
- [ ] third

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by patriciarussellCAS** [comment]
@BCerki @zoeyli-46 @andrea-williams  - We have this old ticket about what happens when a user changes their BCeID name. 
I'm wondering if we should expand this ticket to include the recent issue regarding the user who changed their email address in their BCeID account, which is causing issues. 
Let me know if that makes sense or if not, I can create a new bug ticket for that issue!

**Comment by BCerki** [comment]
Yes, we could add the email change to this ticket. I think it would be updated the same way as the business_bceid_name

**Comment by patriciarussellCAS** [comment]
From refinement: should we update their record every time they log in?

**Comment by patriciarussellCAS** [comment]
Pulling into 2025 Priorities so we can discuss potential scenarios - for example, what happens if a user changes their Operator Name in BCIERS - will that impact anything given it's now different than the BCeID name? If there are no impacts we can de-prioritize this.

**Comment by zoeyli-46** [comment]
If an operator gets bought out by another and has a name change, I wonder if they'll have a new business BCeID? And if they use a new one they may want the old operator information/operations/facilities reports etc. transferred over
